libcaf_io/src/io/basp/instance.cpp

```diff
@@ -312,8 +322,13 @@ connection_state instance::handle(execution_unit* ctx, connection_handle hdl,
         return serializing_basp_payload_failed;
       }
       // Check the application ID.
-      auto whitelist = get_or(config(), "middleman.app-identifiers",
-                              defaults::middleman::app_identifiers);
+      string_list whitelist;
+      if (auto lst = get_if<string_list>(&config(),
+                                         "middleman.app-identifiers")) {
+        whitelist = std::move(*lst);
+      } else {
+        whitelist.emplace_back(to_string(defaults::middleman::app_identifier));
+      }
```

Comment:
The curly braces are here because the if is broken into two lines?

Reply:
No particular reason. I've removed the braces.